src/main.js

```diff
@@ -205,8 +233,8 @@ function hideMainPoster() {
   mainPoster.classList.add("hidden"); 
 }
 function showSavedPosters() {
-  savedPosters.classList.remove("hidden");
+  savedPostersDisplay.classList.remove("hidden");
```

Comment:
Potentially look into using a for loop to look at each of your querySelected elements (may want to pass them into your function as an array). In the loop block take each of the elements and add or remove the `hidden` property that is respective to what you want to display or take away, i.e. if you are going back to the main poster section from say saved posters, after the home button click, then you'll want to hide the home button and hide the saved view section, while simultaneously you'll need to unhide the  `mainPoster` section and the save poster, generate random, and view saved buttons.

Reply:
@cameronRomo Thank you for the helpful advice!  Will look into that now.